FluentTerminal.App.ViewModels/MainViewModel.cs

```diff
@@ -459,18 +482,14 @@ private void Terminal_CustomTitleChanged(object sender, string e)
 
         private void Terminal_ShellTitleChanged(object sender, string e)

```

Comment:
remove stub or use this one instead of the more generic propertychanged again (the custom event is more readable in my opinion)

Reply:
Huh, I've just removed `CustomTitleChanged` event. Not sure why you think that the custom event is more reliable? `PropertyChanged` fires anyway, and covers both `ShellTitleChanged` and `CustomTitleChanged`.
I've removed the stub in one of my following PRs.

Reply:
No big difference (performance or anything) between `PropertyChanged` and custom events, but I believe that the code is simpler (two `event`s removed, subscribe/unsubscribe, different handlers...)

Reply:
You can see all the benefits in my following PR https://github.com/felixse/FluentTerminal/pull/604